metakb/transform/civic.py

```diff
@@ -0,0 +1,837 @@
+                ]
+            else:
+                methods = []
+        return methods
+
+    def _get_eid_documents(self, source):
+        """Get an EID's documents.
+
+        :param dict source: An evidence item's source
+        """
+        documents = None
+        source_type = source['source_type'].upper()
+        if source_type in schemas.SourcePrefix.__members__:
+            prefix = schemas.SourcePrefix[source_type].value
+            document_id = f"{prefix}:{source['citation_id']}"
+            xrefs = []
+            if source['asco_abstract_id']:
+                xrefs.append(f"asco.abstract:{source['asco_abstract_id']}")
+            if source['pmc_id']:
+                xrefs.append(f"pmc:{source['pmc_id']}")
+
+            documents = schemas.Document(
+                id=document_id,
+                label=source['citation'],
+                description=source['name'],
+                xrefs=xrefs if xrefs else None
+            ).dict(exclude_none=True)
+        else:
+            logger.warning(f"{source_type} not in schemas.SourcePrefix.")
+        return [documents]
+
+    def _get_aid_documents(self, assertion, propositions_documents_ix):
+        """Get an AID's documents.
+
+        :param dict assertion: A CIViC Assertion
+        :param propositions_documents_ix: Keeps track of proposition and
+            documents indexes
+        :return: A list of AID documents
+        """
+        # NCCN Guidlines
+        label = assertion['nccn_guideline']
+        version = assertion['nccn_guideline_version']
+        document_id = '_'.join((label + version).split())
+        document_ix = \
+            self._set_ix(propositions_documents_ix, 'documents',
+                         document_id)
+        documents = list()
+        documents.append(schemas.Document(
+            id=f"document:{document_ix:03}",
+            document_id="https://www.nccn.org/professionals/"
+                        "physician_gls/default.aspx",
+            label=f"NCCN Guidelines: {label} version {version}"
+        ).dict(exclude_none=True))
+
+        # TODO: Check this after first pass
+        # ACMG Codes
+        if assertion['acmg_codes']:
+            for acmg_code in assertion['acmg_codes']:
+                document_id = f"acmg:{acmg_code['code']}"
+                documents.append(schemas.Document(
+                    id=document_id,
+                    label=acmg_code['code'],
+                    description=acmg_code['description']
+                ).dict(exclude_none=True))
+
+        return documents
+
+    def _get_record(self, record_id, records):
+        """Get a CIViC record by ID.
+
+        :param str record_id: The ID of the record we are searching for
+        :param dict records: A dict of records for a given CIViC record type
+        """
+        for r in records:
+            if r['id'] == record_id:
+                return r
+
+    def _add_to_list(self, eid, key, list_name):
```

Comment:
I don't see this method has been used, could be removed if not.

Reply:
Good catch! Removing now.